src/System.Private.CoreLib/shared/System/Number.Formatting.cs

```diff
@@ -1460,9 +1457,471 @@ internal static unsafe char ParseFormatSpecifier(ReadOnlySpan<char> format, out
                 '\0';
         }
 
-        internal static unsafe void NumberToString(ref ValueStringBuilder sb, ref NumberBuffer number, char format, int nMaxDigits, NumberFormatInfo info)
+        internal static unsafe void IntegerBufferToString(ref ValueStringBuilder sb, ref IntegerBuffer integer, char format, int nMaxDigits, NumberFormatInfo info)
```

Comment:
@jkotas, the bulk of the "new lines" comes from duplicating `BufferToString` and `BufferToStringFormat`.

I'll think about this some more and see if we can reduce this duplication.